devito/types/sparse.py

```diff
@@ -535,6 +536,27 @@ def _point_symbols(self):
         return tuple(Scalar(name='p%s' % d, dtype=self.dtype)
                      for d in self.grid.dimensions)
 
+    @cached_property
+    def _position_map(self):
+        """
+        Symbols map between grid origin and sparse point position
```

Comment:
"Symbolic map between the grid origin and sparse point position(s)..."?

Reply:
I prefer what I had, this sounds like it maps `{ origin : coordinate}` which it's something else.

Reply:
reverted